### PR TITLE
VBE Vertex 2.4.0

### PR DESCRIPTION
**Pull request by leslietilling**
## Purpose of this pull request

This pull request (PR) updates content for redesigned Vertex UI in Magento for 2.4.0.

Internal staging build: 530

If Magento Commerce, please specify:

- [ ] Commerce only?
- [ ] Commerce with B2B?

## Affected documentation pages

- https://docs.magento.com/user-guide/configuration/sales/tax.html
- https://docs.magento.com/user-guide/configuration/sales/address-validation.html (renamed)
- https://docs.magento.com/user-guide/tax/vertex.html
- https://docs.magento.com/user-guide/tax/vertex-configure-magento.html
- https://docs.magento.com/user-guide/tax/vertex-configure-address.html

whatsnew
The updated [Vertex extension](https://docs.magento.com/user-guide/tax/vertex.html) includes a redesigned UI for [configuration](https://docs.magento.com/user-guide/tax/vertex-configure-magento.html) and a change from _address validation_ to [_address cleansing_](https://docs.magento.com/user-guide/tax/vertex-configure-address.html) terminology.

### CODE REVIEW

File: src/tax/vertex-configure-address.md [hunk 1]
```diff
@@ -1,36 +1,38 @@
...
 1. To define the [scope]({% link configuration/scope.md %}) of the configuration for a multisite or multistore installation, set the **Store View** chooser in the upper-left corner to the specific website or store view that is to be configured.
 
-1. In the left panel, expand **Sales** and choose **Address Validation**.
+1. In the left panel, expand **Sales** and choose **Address Cleansing**.
 
-1. Set **Use Vertex Address Validation** to `Enable`.
+1. Set **Vertex Address Cleansing** to `Enable`.
 
-   This setting enables allows you to configure the address validation settings.
+   This setting enables allows you to configure the address cleansing settings.
```
Comment (ccondra) [inline src/tax/vertex-configure-address.md:24]: extra word
```suggestion
   This setting enables you to configure the address cleansing settings.
```

File: src/configuration/sales/tax.md [hunk 2]
```diff
@@ -71,10 +45,31 @@ Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Sale
...
-## Vertex Logging
+### Delivery Terms
+
+![]({% link images/images/config-sales-tax-vertex-delivery-terms.png %}){: .zoom}
+_Vertex - Delivery Terms_
+
+|Field|[Scope]({% link configuration/scope.md %})|Description|
+|--- |--- |--- |
+|Global Delivery Term|Store View|Identifies the global Delivery Terms for Vertex taxes on purchases and shipments. The terms provide about seventeen options. Options: <br/>**CFR** - Cost and Freight <br/>**CIF** - Cost Insurance and Freight <br/>**CIP** - Carriage Insurance Paid To <br/>**CPT** - Carriage Paid To <br/>**CUS** - Customer <br/>**DAF** - Delivered at Frontier <br/>**DAP** - Delivered at Place <br/>**DAT** - Delivered at Terminal <br/>**DDP** - Delivery Duty Unpaid <br/>**DEQ** - Delivered Ex Quay Duty Unpaid <br/>**DES** - Delivered Ex-Ship <br/>**EXW** - Ex Works <br/>**FAS** - Free Along Ship <br/>**FCA** - Free Carrier <br/>**FOB** - Free Onboard Vessel <br/>**SUP** - Supplier Ships (default)|
+|Delivery Term Override|Store View|Identifies rules for overriding the Delivery Terms for a specific country. The actions option is delete. To edit, modify the terms and save the configuration.|
```
Comment (hickskenh) [inline src/configuration/sales/tax.md:56]: determines?  or defines?

File: src/tax/vertex-configure-address.md [hunk 3]
```diff
@@ -1,36 +1,38 @@
 ---
-title: Configuring Storefront Address Validation
+title: Configuring Storefront Address Cleansing
 ---
 
-Your [Vertex Cloud account]({% link tax/vertex-account.md %}) also includes the Vertex Address Validation module. When this functionality is enabled for your Magento store, the storefront prompts the customer to correct the address information on both the shipping and billing steps of the one-page checkout and when the customer adds an address to their account.
+Your [Vertex Cloud account]({% link tax/vertex-account.md %}) also includes the Vertex Address Cleansing module. When this functionality is enabled for your Magento store, the storefront prompts the customer to correct the address information on both the shipping and billing steps of the one-page checkout and when the customer adds an address to their account.
```
Comment (hickskenh) [inline src/tax/vertex-configure-address.md:5]: does this mean vertex compares the entered address against known post office addresses to populate a suggested address?

File: src/tax/vertex-configure-address.md [hunk 4]
```diff
@@ -1,36 +1,38 @@
...
 
 1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.
 
 1. To define the [scope]({% link configuration/scope.md %}) of the configuration for a multisite or multistore installation, set the **Store View** chooser in the upper-left corner to the specific website or store view that is to be configured.
 
-1. In the left panel, expand **Sales** and choose **Address Validation**.
+1. In the left panel, expand **Sales** and choose **Address Cleansing**.
 
-1. Set **Use Vertex Address Validation** to `Enable`.
+1. Set **Vertex Address Cleansing** to `Enable`.
```
Comment (hickskenh) [inline src/tax/vertex-configure-address.md:22]: does Address Cleansing need to be capped as a feature throughout?

File: src/tax/vertex-configure-address.md [hunk 5]
```diff
@@ -1,36 +1,38 @@
...
 
    ![]({% link images/images/config-sales-address-validation.png %}){: .zoom}
-   [_Address Validation settings - enabled_]({% link configuration/sales/address-validation.md %})
+   [_Address Cleansing settings - Enabled_]({% link configuration/sales/address-cleansing.md %})
 
-1. Verify and accept the default value for **Vertex Address Validation API URL**.
+1. Verify and accept the default value for **Address Validation API URL**.
 
-   This connects the integration with Vertex Cloud. The value should match the URL displayed in your Vertex Connectors page for the Magento connector in the Address Lookup URL field.
+   This connects the integration with Vertex Cloud. The value should match the URL displayed in your _Vertex Connectors_ page for the Magento connector in the _Address Lookup URL_ field.
```
Comment (hickskenh) [inline src/tax/vertex-configure-address.md:31]: these field names are ital.  review for other field names for consistency

File: src/tax/vertex-configure-magento.md [hunk 6]
```diff
@@ -6,109 +6,101 @@ redirect_from:
...
+   - Enter the **Location Code** from the _Registration Details_ section of your Vertex Cloud account.
 
-Set the delivery terms (shipping terms) that determines the vendor or customer jurisdiction where the title transfer of a supply takes place. You can set a global term and overrides per country. These options provide support to EU VAT.
+      This is the specific jurisdiction where the company is registered to file tax returns. This value corresponds to the Location Code that is entered as a Registered Location.
 
-1. Expand ![]({% link images/images/btn-expand.png %}) the **Vertex Delivery Terms** section.
+   - If you want to specify a seller address separately from the shipping origin, set **Use Shipping Origin as Source** to `No` and complete the address fields.
 
-1. To change the default **Global Delivery Term**, select an option (`SUP - Supplier Ships` is the default).
+      If you leave the default `Yes` setting, Vertex uses the address set in the Origin section of the Shipping Settings.
```
Comment (hickskenh) [inline src/tax/vertex-configure-magento.md:77]: does shipping settings have more info in another topic?   add link?

File: src/tax/vertex.md [hunk 7]
```diff
@@ -4,51 +4,25 @@ title: Vertex Cloud
 
 Vertex is a cloud-based solution that automates your sales and use tax compliance, and generates a signature-ready PDF for your monthly returns. To get started, you need a Vertex account, then complete configuration settings.
 
-The integration includes tax configuration for your stores, overrides per country, and performance enhancing options.
+The integration includes tax configuration and address verification for your stores, overrides per country, and performance enhancing options.
```
Comment (hickskenh) [inline src/tax/vertex.md:7]: is it verification or cleansing?

File: src/tax/vertex.md [hunk 8]
```diff
@@ -4,51 +4,25 @@ title: Vertex Cloud
...
-
-- {:.new}New Feature — **Flexible fields for custom taxability drivers**
-
-   The updated Vertex extension includes support for custom tax drivers that you can use to include additional information needed to calculate tax in more complex tax situations. These drivers are passed in configurable Vertex Flexible Fields that support Code, numeric, and date values.
-
-   For more information, see [Vertex Flexible Fields]({% link tax/vertex-configure-magento.md %}#flex-fields).
+   - Configuration in the Magento Admin has been redesigned to be more user friendly
+   - Vertex can now be configured to use the Shipping Origin address as the Seller in Vertex Tax calls
+   - _Address Validation_ in the configuration has been renamed to _Address Cleansing_
+   - Improvements for calculations on an order with both physical and virtual products before a billing address was set
```
Comment (hickskenh) [inline src/tax/vertex.md:22]:
```suggestion
   - Improvements have been made for calculations on an order with both physical and virtual products before a billing address was set
```

File: src/tax/vertex.md [hunk 9]
```diff
@@ -4,51 +4,25 @@ title: Vertex Cloud
...
+   - Improvements for updating the billing address with Vertex's recommendation
```
Comment (hickskenh) [inline src/tax/vertex.md:23]:
```suggestion
   - Improvements have been made for updating the billing address with Vertex's recommendation
```

File: src/tax/vertex.md [hunk 10]
```diff
@@ -4,51 +4,25 @@ title: Vertex Cloud
...
+   - Improvements for bundle and virtual products tax calculations
```
Comment (hickskenh) [inline src/tax/vertex.md:24]:
```suggestion
   - Improvements have been made for bundle and virtual products tax calculations
```

File: src/tax/vertex.md [hunk 11]
```diff
@@ -4,51 +4,25 @@ title: Vertex Cloud
...
+   - Updates to address validation/cleansing display to better support Internet Explorer 11
```
Comment (hickskenh) [inline src/tax/vertex.md:25]:
```suggestion
   - Address validation/cleansing has been updated display to better support Internet Explorer 11
```

Comment (hickskenh) [reply src/tax/vertex.md:25]: also, i thought we weren't supporting IE 11.

File: src/configuration/sales/address-cleansing.md [hunk 12]
```diff
@@ -0,0 +1,18 @@
...
+![]({% link images/images/config-sales-address-validation.png %}){: .zoom}
+[_Vertex Address Cleansing settings - enabled_]({% link tax/vertex-configure-address.md %})
+
+{:.bs-callout-info}
+Using the storefront address cleansing requires that you first [configure the Vertex Tax Calculations]({% link tax/vertex-configure-magento.md %}) to connect to your Vertex Cloud account.
+
+|Field|[Scope]({% link configuration/scope.md %})|Description|
+|--- |--- |--- |
+|Vertex Address Cleansing|Store|Enables Vertex Address Validation to prompt correction of the address information on both the shipping and billing steps of the one-page checkout. Options: Enable / Disable|
+|Address Validation API URL|Store|The URL that is used to validate the connection between Magento and Vertex Cloud address services. This value should match the URL displayed in your Vertex Connectors page for the Magento connector in the Address Lookup URL field. Default value: `https://mgcsconnect.vertexsmb.com/vertex-ws/services/LookupTaxAreas70`|
```
[the remark this answers is not in the document]
Comment (leslietilling) [reply src/configuration/sales/address-cleansing.md:16]: We should probably do that here.

**Comment by leslietilling** [comment]
running tests

**Pull request merged**